nginx-controller/nginx/ingress.go

```diff
@@ -15,6 +15,32 @@ type IngressEx struct {
 }
 
 type MergeableIngresses struct {
-	Master *IngressEx
+	Master  *IngressEx
 	Minions []*IngressEx
 }
+
+var mergeableIngressAnnotationBlacklist = map[string][]string{
```

Comment:
I think it is better to have two maps of map[string]bool for two black lists -- the master one and the minion one.

Reply:
To keep it simple, can we just add a `[]string`